src/NzbDrone.Core/ImportLists/Simkl/SimklParser.cs

```diff
@@ -43,7 +43,7 @@ public virtual IList<ImportListItemInfo> ParseResponse(ImportListResponse import
                 {
                     var tentativeTvdbId = int.TryParse(show.Show.Ids.Tvdb, out var tvdbId) ? tvdbId : 0;
 
-                    if (tentativeTvdbId > 0 && show.AnimeType == SimklAnimeType.Tv)
+                    if (tentativeTvdbId > 0 && (show.AnimeType == SimklAnimeType.Tv || show.AnimeType == SimklAnimeType.Ona || show.AnimeType == SimklAnimeType.Ova || show.AnimeType == SimklAnimeType.Special))
```

Comment:
With C# 9 you can now do the following which makes it a little easier to read and add more options if the need arises.

```suggestion
                    if (tentativeTvdbId > 0 && (show.AnimeType is SimklAnimeType.Tv or SimklAnimeType.Ona or SimklAnimeType.Ova or SimklAnimeType.Special))
```